examples/apps/generic_client/generic_client.py

```diff
@@ -166,7 +166,7 @@ def Main(args=None):
                 " *****************")
 
     global direct_jrpc
-    direct_jrpc = DirectJsonRpcApiConnector(config_file=None, config=config)
+    direct_jrpc = AvalonDirectClient(config_file=None, config=config)
```

Comment:
What's difference between config_file and config. Please add a comment in code..

Reply:
Comments are there in AvalonDirectClient